### ISSUE TITLE
Benchmark script instructions are out of date

### ISSUE THREAD

**Issue by gaearon** [emotion-js/emotion#857]
Not that they're meant to be consumed, but none of the instructions https://github.com/emotion-js/emotion/tree/master/scripts/benchmarks are actually working.

```
open ./packages/benchmarks/dist/index.html
```

^^ this path doesn't exist, they're in a different folder

```
yarn compile --watch
yarn benchmarks --watch
```

^^ there are no such commands either in top level or in the benchmark folder

I did manage to get it running though. Just flagging.

**Comment by mitchellhamilton** [comment]
Thanks for the issue, fixed via 97e7e6389471586934bccbe3cd68bdf576a50727. BTW, you can try the benchmarks at https://benchmarks.emotion.sh

**Comment by gaearon** [comment]
To be clear, I was actually using them to measure some React performance changes so I need to run them locally :-)
